README.md

```diff
@@ -16,7 +16,7 @@ $ composer require php-http/guzzle7-adapter
 
 ## Documentation
 
-Please see the [official documentation](http://docs.php-http.org/en/latest/clients/guzzle6-adapter.html).
+Please see the [official documentation](http://docs.php-http.org/en/latest/clients/guzzle7-adapter.html).
```

Comment:
💡 This URL returns a 404 for now, but I assume that an updated documentation will be pushed here as soon as it's available?

Reply:
Ah, leaving the six here was apparently intentional (#1). Reverted.